app/assets/javascripts/metrics.js

```diff
@@ -41,10 +44,13 @@ function formatSeconds(s) {
     return output.join(' ');
 }
 
+function formatDate(date) {
+    var months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
```

Comment:
You're not going to like this, but can you stick with single quotes through this file please? So that JS Lint passes :see_no_evil: